templates/nextcloud.nomad

```diff
@@ -0,0 +1,80 @@
+job "nextcloud" {
+  datacenters = ["dc1"]
+  type = "service"
+  
+  group "nc" {
+    task "nextcloud" {
+      driver = "docker"
+      config {
+        image = "liquidinvestigations/liquid-nextcloud"
+        volumes = [
+          "${liquid_volumes}/nextcloud/nextcloud:/var/www/html",
+          "${liquid_collections}/uploads/data:/var/www/html/data/uploads/files",
+        ]
+        args = ["/bin/sh", "-c", "chown -R 33:33 /var/www/html/ && echo chown done && /entrypoint.sh apache2-foreground"]
+        port_map {
+          http = 80
+        }
+        labels {
+          liquid_task = "nextcloud"
+        }
+      }
+      template {
+        data = <<EOF
+            {{- range service "nextcloud-pg" }}
+              NEXTCLOUD_POSTGRES_HOST = {{.Address}}:{{.Port}}
+            {{- end }}
+            NEXTCLOUD_HOST = nextcloud.{{ key "liquid_domain" }}
+            NEXTCLOUD_ADMIN_USER="admin"
+            NEXTCLOUD_ADMIN_PASSWORD="admin"
+            NEXTCLOUD_POSTGRES_DB="nextcloud"
+            NEXTCLOUD_POSTGRES_USER="postgres"
+            NEXTCLOUD_POSTGRES_PASSWORD="secret"
+            OC_PASS="a_very_good_secret"
```

Comment:
What does this value do? If it's supposed to be a random secret like Django's `SECRET_KEY` then we can auto-generate it and store it in Vault - take a look at hoover-search and see how it's done there.

Reply:
Yeah it is, I will look at it and change it.